hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/impl/producers/formats/v7/BlockRecordFormatV7.java

```diff
@@ -165,12 +165,13 @@ private void writeHeader(
      * @param outputStream the output stream to write to
      * @param item the item to extract/convert
      */
-    private void writeRecordStreamItem(
+    private void writeRecordStreamItem( // TODO: should we remove this unused method?
```

Comment:
We're still making changes to writing records, so I'd suggest we leave the unused methods in this class for now. Maybe turn sonar off for them if you'd like